includes/access.php

```diff
@@ -1837,54 +1837,56 @@ function pods_access_get_capabilities_preview( string $pod_type, string $pod_nam
  * @return array The pod settings config for access-related settings.
  */
 function pods_access_settings_config(): array {
+	$did_init = doing_action( 'init' ) || did_action( 'init' );
```

Comment:
Lets add a comment why this is done for future reference.

```suggestion
	// Only use translation functions after `init` to prevent a WP core notice.
	$did_init = doing_action( 'init' ) || did_action( 'init' );
```